Packs/CortexXDR/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Palo Alto Networks Cortex XDR - Investigation and Response",
     "description": "Automates Cortex XDR incident response, and includes custom Cortex XDR incident views and layouts to aid analyst investigations.",
     "support": "xsoar",
-    "currentVersion": "3.0.27",
+    "currentVersion": "3.1.0",
```

Comment:
new integration should bump version by major

Reply:
Done